### ISSUE TITLE
Token Screen blanks out when clicked

### ISSUE THREAD

**Issue by xxmafiaxxx** [workbenchapp/solana-workbench#246]
## Expected Behavior
should see wallets

## Current Behavior
black screen

## Steps to Reproduce
click on tokens

**Comment by SvenDowideit** [comment]
yeah, there's some weird stuff that got lost in dev - I _think_ #241 now resolves them - if you're willing to test that \o/!

**Comment by xxmafiaxxx** [comment]
ill check it now

**Comment by SvenDowideit** [comment]
and we're released https://github.com/workbenchapp/solana-workbench/releases/tag/v0.4.0 - tho with app signing issues, and the token detail data in the accordions are black text on black - something for v0.4.1 in a day or two
